java/src/test/java/org/rocksdb/ReadOnlyTest.java

```diff
@@ -31,81 +31,51 @@ public void readOnlyOpen() throws RocksDBException {
          final RocksDB db = RocksDB.open(options,
              dbFolder.getRoot().getAbsolutePath())) {
       db.put("key".getBytes(), "value".getBytes());
-      try (final RocksDB db2 = RocksDB.openReadOnly(
-          dbFolder.getRoot().getAbsolutePath())) {
-        assertThat("value").
-            isEqualTo(new String(db2.get("key".getBytes())));
+    }
+    try (final RocksDB db = RocksDB.openReadOnly(dbFolder.getRoot().getAbsolutePath())) {
+      assertThat("value").isEqualTo(new String(db.get("key".getBytes())));
+    }
+
+    final ColumnFamilyOptions cfOpts = new ColumnFamilyOptions();
```

Comment:
This is never closed.

Reply:
It's weird to close an option object. What kind of resource can't the `finalize` close?

Reply:
Why isn't `ColumnFamilyDescriptor` closed explicitly?

Reply:
It's weird to close an option object, but I understand the motivation behind this to free the memory asap. https://github.com/facebook/rocksdb/blob/master/java/src/main/java/org/rocksdb/ColumnFamilyDescriptor.java#L24, it's not closed explicitly here.